README.md

```diff
@@ -191,7 +191,7 @@ Silent mode off: `.o-ft-buttons--inverse`
 ### Deprecated
 
 The `--faux` modifier is now deprecated and will be removed in next major release.
-The `.o-ft-buttons__pagination` wasn't following the BEM convention properly and will be removed in next major release. Use `..o-ft-buttons-pagination` instead.
+The `.o-ft-buttons__pagination` wasn't following the BEM convention properly and will be removed in next major release. Use `.o-ft-buttons-pagination` instead.
```

Comment:
Alright, here it is: 87e5d96

That's going to be a lot of code. I'll release a major version right after it with much leaner code.